packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -9,7 +9,10 @@ import type Dispatcher from '../../dispatcher'
 import Ceramic, { CeramicConfig } from "../../ceramic"
 
 const DID_MATCHER = '^(did:([a-zA-Z0-9_]+):([a-zA-Z0-9_.-]+(:[a-zA-Z0-9_.-]+)*)((;[a-zA-Z0-9_.:%-]+=[a-zA-Z0-9_.:%-]*)*)(/[^#?]*)?)([?][^#]*)?(#.*)?';
-const CHAIN_ID = 'inmemory:12345'
+
+// In-memory CAS is used in testing only and will not be included if there are other anchoring services. That's why
+// its chain ID is equal to Ropsten chain ID.
+export const IN_MEMORY_ANCHOR_SERVICE_CHAIN_ID = 'eip155:3'
```

Comment:
This doesn't make sense. The in memory anchor service should have it's own unique chainID otherwise it may cause confusion or even bugs / other issues.

Reply:
This "anchoring" service is used just for testing. It was never meant to be used in production.